src/mscorlib/corefx/System/Globalization/SortKey.cs

```diff
@@ -52,26 +52,6 @@ internal SortKey(String localeName, String str, CompareOptions options, byte[] k
             _options    = options;
             _string   = str;
         }
-
-        [OnSerializing]
```

Comment:
Yes, rewriter was stripping some of them. I found all the ones getting stripped and reverted them, also opened this to add tests https://github.com/dotnet/corefx/issues/15731